graylog2-plugin-interfaces/src/test/java/org/graylog2/plugin/BaseConfigurationTest.java

```diff
@@ -44,6 +44,9 @@
         @Parameter(value = "rest_listen_uri", required = true)
         private URI restListenUri = URI.create("http://127.0.0.1:12900/");
 
+        @Parameter(value = "web_listen_uri", required = true)
+        private URI webListenUri = URI.create("http://127.0.0.1:9000/");
```

Comment:
This is non-trivial, because the URL schemes of the REST API and the web interface clash. So this would mean that we would either have them a) live on two separate namespaces (like it is now) or b) invalidate all bookmarks that people might have created to the web interface.